src/geneva/Go2.cpp

```diff
@@ -823,8 +823,10 @@ void Go2::parseCommandLine(
 			("client", "Indicates that this program should run as a client or in server mode. Note that this setting will trigger an error unless called in conjunction with a consumer capable of dealing with clients")
 			("maxClientDuration", po::value<std::string>(&maxClientDuration)->default_value(EMPTYDURATION),
 				R"(The maximum runtime for a client in the form "hh:mm:ss". Note that a client may run longer as this time-frame if its work load still runs. The default value "00:00:00" means: "no time limit")")
-			("consumer,c", po::value<std::string>(&m_consumer_name)->default_value("stc"), consumer_help.str().c_str());
-
+			("consumer,c", po::value<std::string>(&m_consumer_name)->default_value("stc"), consumer_help.str().c_str())
+       		        ("ioc,i", po::value<int>(&m_ioc)->default_value(0),
+		         "io_per_cpu (network based consumers only");
```

Comment:
I suggest a better description what this is for, or where to find information. Also ) missing